package/yast2-network.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Oct 21 11:01:04 UTC 2016 - kanderssen@suse.com
+- Use '2' as the default value for forward delay in bridges created
+  by the virt proposal (bsc#988278)
```

Comment:
I will close it by now until more information is given, as the default not enable STP and in that case seems that the forwarddelay is not taking in cosideration, and I thought it could be independent of STP. The kernel min value is '2' although the IEEE range starts in 4.

More info here

https://github.com/openSUSE/wicked/blob/master/src/bridge.c#L293-L356

And the bug has been also updated by @mtomaschewski . Thnx @mtomaschewski !!